### PR TITLE
Kubernetes dev

### PR DESCRIPTION
**Pull request by poornima-metron**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,9 +38,10 @@
   "dependencies": {
     "@jupiterone/integration-sdk-dev-tools": "^8.22.6",
```
Comment (suraj-metron) [inline package.json:39]: Please update all packages to latest version. Check the armis for reference

**Review by suraj-metron — CHANGES_REQUESTED** [review]

File: src/steps/cron-jobs/index.ts [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import {
   createDirectRelationship,
+  getRawData,
+  IntegrationMissingKeyError,
   IntegrationStep,
   RelationshipClass,
 } from '@jupiterone/integration-sdk-core';
+import * as k8s from '@kubernetes/client-node';
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/steps/cron-jobs/index.ts]: imported only relevant

File: src/steps/daemon-sets/index.ts [hunk 3]
```diff
@@ -1,12 +1,16 @@
 import {
   createDirectRelationship,
+  getRawData,
+  IntegrationMissingKeyError,
   IntegrationStep,
   RelationshipClass,
 } from '@jupiterone/integration-sdk-core';
+import * as k8s from '@kubernetes/client-node';
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/steps/daemon-sets/index.ts]: imported only relevant

File: src/steps/jobs/index.test.ts [hunk 4]
```diff
@@ -73,7 +78,18 @@ describe('#fetchJobs', () => {
...
+                _class: { const: RelationshipClass.HAS },
+                _type: { const: 'kube_container_spec_has_job' },
+              },
+            },
+          },
+        },
       ],
     });
-  });
+  }, 500000);
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/steps/jobs/index.test.ts:94]: removed as not needed

**Review by suraj-metron — APPROVED** [review]

**Pull request merged**